parm/config/config.base.emc.dyn

```diff
@@ -293,6 +293,11 @@ export lobsdiag_forenkf=".true."
 export DO_GLDAS="YES"
 export gldas_cyc=00
 
+# Exception handling that when DO_GLDAS is set, the FHOUT must be 1
+if [ $DO_GLDAS = "YES" ]; then
+  export FHOUT=1
```

Comment:
Please add a comment up on L227 where this is originally set that the value may be overridden if GLDAS is on.

Reply:
It is done.